### DISCUSSION TITLE
Failure to Reset EEPROM using USB cable

### DISCUSSION THREAD

**Discussion by reginaldStjohn** [bitcraze/discussions#941] [Q&A] [answered]
Following [these ](https://wiki.bitcraze.io/doc:crazyflie:dev:starting#reset_eeprom) instructions to reset the eeprom using the usb cable we get a failure.

The failure comes from the script trying to enumerate the radio, which is unplugged for this step, and fails. 
```
Connecting to radio://0/80/2M/E7E7E7E7E7
ERROR:cflib.crazyflie:Couldn't load link driver: Cannot find a Crazyradio Dongle

Traceback (most recent call last):
  File "/home/bitcraze/.local/lib/python3.8/site-packages/cflib/drivers/crazyradio.py", line 124, in __init__
    device = _find_devices()[devid]
IndexError: list index out of range

```
If we plug the radio in while running the script then it tries to connect using the radio, which fails.

We updated our crazyflie to NRF 51 (2023.07 firmware) and there is no change to the error.

**Comment by krichardsson** [comment, marked as answer]
There should usually not be a need to reset the eeprom, the instructions you found on the wiki are very very old. If the problem is that you want to change the radio channel/address, you should probably focus on your other discussion (#939 instead) and change the settings from the client.

If you really want to run the script, it looks like the problem is that python script can not find the Crazyradio. I assume you are using the VM (as in #939)? 
You have to make sure the VM has access to the Crazyradio via USB. Assuming you are using VirtualBox (lots of assumptions today...), right-click the USB connector at the bottom of the VM window and tick `Bitcraze AB Crazyradio 2.0`.

**Comment by reginaldStjohn** [comment]
We wanted to reset the eeprom because we did not know the crazyflie addess, it had been changed, and so we could not connect with the radio.

Thank you for your help. We were able to retrieve the address by connecting using a VM with the 2023.10 VM image. 

However, as mentioned in my other, related post, we are unable to run the VM Image in our VM Host of choice, VMWare, because we cannot get the networking to work.

Sorry about the "old" instructions.  Maybe they could be updated or deleted if they are no longer of use?

**Reply by knmcguire** [reply]
The message on the top of the wiki page you used indicated that the page has been deprecated. This is the new getting started documentation: https://www.bitcraze.io/documentation/tutorials/getting-started-with-development/

**Comment by knmcguire** [comment]
This discussion is also related to this https://github.com/orgs/bitcraze/discussions/939. Also, don't close discussions as they disappear from the list. Rather select an answer that made you solve the problem.

**Reply by knmcguire** [reply]
thanks!
